packages/x-charts/src/hooks/useInteractionItemProps.ts

```diff
@@ -6,7 +6,10 @@ import { UseChartHighlightSignature } from '../internals/plugins/featurePlugins/
 import { UseChartInteractionSignature } from '../internals/plugins/featurePlugins/useChartInteraction';
 
 const onPointerDown = (event: React.PointerEvent) => {
-  if (event.currentTarget.hasPointerCapture(event.pointerId)) {
+  if (
+    'hasPointerCapture' in event.currentTarget &&
```

Comment:
Is this because JSDOM doesn't have `hasPointerCapture`? Why not mock it in the test setup if it only applies to tests?

Reply:
This also solves the issue for users testing our app in jsdom